bin/subs.py

```diff
@@ -225,16 +225,23 @@ def get_requested_metrics(resource):
     if len(tags) == 0:
         return None
 
-    # if no metrics tag, skip it
-    metrics_tag = tags.get('Metrics', tags.get('metrics', None))
+    # retrieve tag configuration
+    tag_key = input_config_dict.get("metrics_tag_key") or 'Metrics'
+    ignore_tag_value = input_config_dict.get("ignore_tag_value") or False
```

Comment:
input_config_dict.get("ignore_tag_value") returns a string. hence always True. This causes "All available metrics" always.